p2p/src/authenticated/discovery/channels.rs

```diff
@@ -36,6 +69,12 @@ impl<P: PublicKey> crate::Sender for Sender<P> {
     /// If a recipient is offline at the time a message is sent, the message will be dropped.
     /// It is up to the application to handle retries (if necessary).
     ///
+    /// # Rate Limiting
+    ///
+    /// If outbound rate limiting is enabled, recipients that exceed their rate limit
```

Comment:
I think we should always enable. There is no reason to ever send above what you are permitted (only can be misconfigured).